usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
@dahorak You can run tests in directory without tests in sub-directories by this command
```
py.test -o norecursedirs='ceph gluster' -svvvx usmqe_tests/api/tendrl/
```